groups/application/sg.tf

```diff
@@ -23,10 +23,9 @@ resource "aws_security_group_rule" "jdbc" {
 
   security_group_id = module.gfn_app_ec2_security_group.this_security_group_id
   description       = "Allow on-premise jdbc queries"
-  for_each          = toset(["1521"])
   type              = "ingress"
-  from_port         = each.value
-  to_port           = each.value
+  from_port         = 1521
```

Comment:
This is an Oracle port but its inbound, I dont think you'll need this rule for Grafana?